src/foundry/client/applications/ui/hotbar.d.mts

```diff
@@ -33,6 +121,12 @@ declare namespace Hotbar {
 
   interface SlotData {
     slot: number;
+    macro: Macro.Implementation | null;
+    key: number;
+    img: Macro.Implementation["img"] | null;
```

Comment:
It's probably fine here but generally I try to avoid `XYZ["prop"]`. It has about the same performance expense as referring to the object as a whole (which is a concern at compile time) and namespacing is generally better. This is overall pretty innocent though.

Reply:
Yeah in _theory_ it'd be `Macro.Img` but because that's so unlikely to be useful I didn't want to ask for it here.

I put the comment mostly to be informative.